.github/workflows/build_hub.yml

```diff
@@ -91,15 +91,18 @@ jobs:
         run: |
           docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite
           
-      - name: Wait for the API to become available
-        uses: cygnetdigital/wait_for_response@v2.0.0
+      - name: Give some time for the container to start
```

Comment:
Besides the gradlew addition, this is the only workflow change, which is switching out an API online check with a sleep timer. Looks good from a DevOps standpoint 👍